scheduler-server/scheduling-dsl-compiler/tsconfig.json

```diff
@@ -95,6 +95,9 @@
 
     /* Completeness */
     // "skipDefaultLibCheck": true,                      /* Skip type checking .d.ts files that are included with TypeScript. */
-    "skipLibCheck": true                                 /* Skip type checking all .d.ts files. */
+    "skipLibCheck": true,                                 /* Skip type checking all .d.ts files. */
   },
+  "exclude": [
+    "./src/libs/scheduler-edsl-fluent-api.ts"  // This file is used at runtime, does not need to be compiled.
```

Comment:
This may not need to be compiled, but we do need it for the type assertion in main.ts. For the command expansion service, we have an additional build step to copy the ts files used at runtime to the build folder. I can add that here too.